platformio.ini

```diff
@@ -15,6 +15,15 @@
 ; development use VSCode to change the target to a non-default
 ; by clicking on the target name in the bottom status bar.
 [platformio]
+; include custom environments to be build
+extra_configs =
+	./CustomDevices/_template/MyCustomDevice_platformio.ini
```

Comment:
The template shouldn't be included, correct?

Reply:
Yeah, will remove it. Also in the all devices environment.

Reply:
Done!